### PR TITLE
Added typography and lists components

### PR DESCRIPTION
**Pull request by vrinda-mahajan**
Added Typography and Lists component. Please review my PR.
Preview
Typography:  https://elegant-rosalind-0c39ab.netlify.app/docs.html#typography
Lists: https://elegant-rosalind-0c39ab.netlify.app/docs.html#lists

### CODE REVIEW

**Review by partha8 — APPROVED** [review]
Looks good!

File: docs.html [hunk 1]
```diff
@@ -783,6 +783,275 @@ <h5 class="component-subheading">Form Example</h5>
...
+        </li>
+    </ul>
+    </div>
+    
+<pre>
+    <code class="html">
+        <ul class="list-non-bullet">
+            <li>
+                <div class="notification flex-r">
+                    Stacked Notification
```
Comment (vrx29) [inline docs.html:1034]: You can reduce this extra div by directly using li as the container and try to give the "notification flex-r' class to li tag.

Comment (vrinda-mahajan) [reply docs.html:1034]: But if I want to put another component (instead of notification) inside stacked lists. Then it won't be reusable.

Comment (vrx29) [reply docs.html:1034]: No issues then. Other things are good you can go ahead and merge

**Review by vrx29 — APPROVED** [review]

File: docs.html (hunk 1 above)
Comment (vrinda-mahajan) [reply docs.html:1034]: Thanks for reviewing.

**Review by foolhardy21 — APPROVED** [review]
looks good to me

**Pull request merged**